assets/immersiverailroading/rolling_stock/freight/spsf_100ton_logcar.json

```diff
@@ -16,7 +16,7 @@
 		"width": 1
 	},
 	    "freight": {
-        "slots": 1,
+        "slots": 2,
         "width": 1,
```

Comment:
Same question here about width

Reply:
when we talked about it months ago the agreement was to make it 2 slots instead of one

Reply:
I'm fine with that, I'm talking about the `"width": 1` line

Reply:
just how it displays the slots in the gui iirc, so it shows side by side and not on top of each other

Reply:
A width of 1 makes me think it would show on top of each other and look a bit silly, but if you tested it and are good with certifying your name on this change, then I guess we'll roll with it as is lol

Reply:
I messed it it and never saw anything change when messing with it  so i left it alone